edx/edx-enterprise
# [ENT-750] Remove inaccurate audit language on landing page.

**Description:** Removes inaccurate audit language on the landing page; just removes everything after the semicolon.

**JIRA:** [ENT-750](https://openedx.atlassian.net/browse/ENT-750)

**Merge deadline:** ASAP

**Testing instructions:**

1. Go to the course enrollment page for a course with both the audit and verified track available. See that the audit track description reads 'Not eligible for a certificate.', and nothing more.
